core/services/relay/evm/chain_reader.go

```diff
@@ -40,10 +40,12 @@ type chainReader struct {
 	bindings
 	codec commontypes.RemoteCodec
 	commonservices.StateMachine
+	isStarted bool
```

Comment:
You should be able to access the started state in the `StateMachine` without the need to add a new property.

Reply:
Thanks for this! Looks cleaner. I was trying to replicate the way [txm components](https://github.com/smartcontractkit/chainlink/blob/v2.14.7-goreleaser-test17/common/txmgr/broadcaster.go#L150) do this

Reply:
changes addressed here: [`be1d2c8` (#14123)](https://github.com/smartcontractkit/chainlink/pull/14123/commits/be1d2c8f4b5d0cffe09eeca3e4f3129bfad3de84)

Reply:
I'll better use `StateMachine.Ready()`. Changes here: [`4ea601d` (#14123)](https://github.com/smartcontractkit/chainlink/pull/14123/commits/4ea601d1717f05d8944a16f2046711d2bda8a200)

Reply:
addressed here: [`19a0268` (#14123)](https://github.com/smartcontractkit/chainlink/pull/14123/commits/19a026846120854ec30bdcdacde7120db2d5637f)